src/main/java/org/symphonyoss/symphony/messageml/elements/Styles.java

```diff
@@ -0,0 +1,151 @@
+    ALLOWED_PROPERTIES.add("box-sizing");
+    ALLOWED_PROPERTIES.add("caption-side");
+    ALLOWED_PROPERTIES.add("clear");
+    ALLOWED_PROPERTIES.add("color");
+    ALLOWED_PROPERTIES.add("content");
+    ALLOWED_PROPERTIES.add("counter-increment");
+    ALLOWED_PROPERTIES.add("counter-reset");
+    ALLOWED_PROPERTIES.add("display");
+    ALLOWED_PROPERTIES.add("empty-cells");
+    ALLOWED_PROPERTIES.add("font");
+    ALLOWED_PROPERTIES.add("font-family");
+    ALLOWED_PROPERTIES.add("font-kerning");
+    ALLOWED_PROPERTIES.add("font-size");
+    ALLOWED_PROPERTIES.add("font-size-adjust");
+    ALLOWED_PROPERTIES.add("font-stretch");
+    ALLOWED_PROPERTIES.add("font-style");
+    ALLOWED_PROPERTIES.add("font-variant");
+    ALLOWED_PROPERTIES.add("font-weight");
+    ALLOWED_PROPERTIES.add("height");
+    ALLOWED_PROPERTIES.add("letter-spacing");
+    ALLOWED_PROPERTIES.add("line-height");
+    ALLOWED_PROPERTIES.add("list-style");
+    ALLOWED_PROPERTIES.add("list-style-image");
+    ALLOWED_PROPERTIES.add("list-style-position");
+    ALLOWED_PROPERTIES.add("list-style-type");
+    ALLOWED_PROPERTIES.add("margin");
+    ALLOWED_PROPERTIES.add("margin-bottom");
+    ALLOWED_PROPERTIES.add("margin-left");
+    ALLOWED_PROPERTIES.add("margin-right");
+    ALLOWED_PROPERTIES.add("margin-top");
+    ALLOWED_PROPERTIES.add("max-height");
+    ALLOWED_PROPERTIES.add("max-width");
+    ALLOWED_PROPERTIES.add("min-height");
+    ALLOWED_PROPERTIES.add("min-width");
+    ALLOWED_PROPERTIES.add("opacity");
+    ALLOWED_PROPERTIES.add("outline");
+    ALLOWED_PROPERTIES.add("outline-color");
+    ALLOWED_PROPERTIES.add("outline-offset");
+    ALLOWED_PROPERTIES.add("outline-style");
+    ALLOWED_PROPERTIES.add("outline-width");
+    ALLOWED_PROPERTIES.add("overflow");
+    ALLOWED_PROPERTIES.add("overflow-x");
+    ALLOWED_PROPERTIES.add("overflow-y");
+    ALLOWED_PROPERTIES.add("padding");
+    ALLOWED_PROPERTIES.add("padding-bottom");
+    ALLOWED_PROPERTIES.add("padding-left");
+    ALLOWED_PROPERTIES.add("padding-right");
+    ALLOWED_PROPERTIES.add("padding-top");
+    ALLOWED_PROPERTIES.add("table-layout");
+    ALLOWED_PROPERTIES.add("text-align");
+    ALLOWED_PROPERTIES.add("text-align-last");
+    ALLOWED_PROPERTIES.add("text-decoration");
+    ALLOWED_PROPERTIES.add("text-decoration-color");
+    ALLOWED_PROPERTIES.add("text-decoration-line");
+    ALLOWED_PROPERTIES.add("text-decoration-style");
+    ALLOWED_PROPERTIES.add("text-indent");
+    ALLOWED_PROPERTIES.add("text-justify");
+    ALLOWED_PROPERTIES.add("text-overflow");
+    ALLOWED_PROPERTIES.add("text-shadow");
+    ALLOWED_PROPERTIES.add("text-transform");
+    ALLOWED_PROPERTIES.add("visibility");
+    ALLOWED_PROPERTIES.add("white-space");
+    ALLOWED_PROPERTIES.add("width");
+    ALLOWED_PROPERTIES.add("word-break");
+    ALLOWED_PROPERTIES.add("word-spacing");
+    ALLOWED_PROPERTIES.add("word-wrap");
+  }
+
+  /**
+   * Validate that the input style attribute
```

Comment:
"Validate that the attributue"... what? I'm dying to know!

Reply:
fixed the mysterious comment